packages/myst-directives/src/blockquote.ts

```diff
@@ -0,0 +1,50 @@
+import type { DirectiveSpec, DirectiveData, GenericNode } from 'myst-common';
+import { normalizeLabel } from 'myst-common';
+import type { Container } from 'myst-spec-ext';
+import classNames from 'classnames';
+
+export const blockQuoteDirective: DirectiveSpec = {
+  name: 'block-quote',
```

Comment:
I think this should be `blockquote` (maybe aliased to `block-quote`?).

Reply:
Hmm. I'm on the fence. I like that it reflects symmetry with `pull-quote`, but of course the internal AST node is `blockquote`. You make the call!